### PR TITLE
Get all positions in radius

### PR DESCRIPTION
**Pull request by Spedwards**
Didn't end up needing this, but it still might be useful.

### CODE REVIEW

File: extensions.js [hunk 1]
```diff
@@ -50,6 +50,22 @@ mod.extend = function(){
             return this._adjacent;
         }
     });
+    Object.defineProperty(RoomPosition.prototype, 'radius', {
+        configurable: true,
+        value: function(radius = 1) {
+            if (radius === 1) return this.adjacent;
+            if (radius < 1) return [];
```
Comment (chobobobo) [inline extensions.js]: This is a filled radius correct?  The analogy doesn't exactly hold up, but if radius=0, wouldn't it make sense to return the roomPosition itself?

File: extensions.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ mod.extend = function(){
         configurable: true,
         value: function(radius = 1) {
             if (radius === 1) return this.adjacent;
-            if (radius < 1) return [];
+            if (radius < 1) return [this];
             const positions = [];
             for (let x = this.x - radius; x < this.x + radius; x++) {
```
Comment (chobobobo) [inline extensions.js]: for radius===1 and pos (1,1) this will loop from 0 - 1, I think you want <= right?

Comment (Spedwards) [inline extensions.js]: No. `< 1` is exactly right. There is a check above for `=== 1` which refers to the property above this one.

Comment (chobobobo) [inline extensions.js]: Ok, bad example, check radius===2, pos 2,2 that will loop from x=0-3, isn't that lopsided?  x=4 should be a part of the radius as well

Comment (Spedwards) [inline extensions.js]: Oh I see what you mean, yeah, it should be `<=` in the for-loops.

**Pull request merged**